tracing-subscriber/src/fmt/mod.rs

```diff
@@ -717,93 +601,51 @@ impl<N, E, F, W> Builder<N, E, F, W> {
     /// [verbosity level]: https://docs.rs/tracing-core/0.1.5/tracing_core/struct.Level.html
     /// [`EnvFilter`]: ../filter/struct.EnvFilter.html
     /// [`with_filter`]: #method.with_filter
-    pub fn with_max_level(self, filter: impl Into<LevelFilter>) -> Builder<N, E, LevelFilter, W> {
+    pub fn with_max_level(
+        self,
+        filter: impl Into<LevelFilter>,
+    ) -> SubscriberBuilder<N, E, LevelFilter, W> {
         let filter = filter.into();
-        Builder {
-            fmt_fields: self.fmt_fields,
-            fmt_event: self.fmt_event,
+        SubscriberBuilder {
             filter,
-            settings: self.settings,
-            make_writer: self.make_writer,
+            inner: self.inner,
         }
     }
 
-    /// Sets the subscriber being built to use a less verbose formatter.
-    ///
-    /// See [`format::Compact`].
-    pub fn compact(self) -> Builder<N, format::Format<format::Compact>, F, W>
```

Comment:
Why did this go away?

Reply:
I think it just moved:
https://github.com/tokio-rs/tracing/blob/b19bc5cc3df8e3f5cda526bb09522a0d7653cc39/tracing-subscriber/src/fmt/mod.rs#L392-L403